closed/OpenJ9.gmk

```diff
@@ -373,6 +373,25 @@ define create_or_update
 	fi
 endef
 
+ifeq (1,$(OPENJ9_USE_CUSTOM_COMPILER))
+  SETUP_CUSTOM_COMPILER_ENV_VARS := \
+	DEVELOPER_DIR_OLD="$(DEVELOPER_DIR)" \
+	CC_OLD="$(CC)" \
+	CXX_OLD="$(CXX)" \
+	DEVELOPER_DIR="$(OPENJ9_DEVELOPER_DIR)" \
+	CC="$(OPENJ9_CC)" \
+	CXX="$(OPENJ9_CXX)" \
+	#
+  CLEANUP_CUSTOM_COMPILER_ENV_VARS := \
```

Comment:
There's no need to capture the old values of these environment variables: they're only set in a sub-shell created by make.

Reply:
@keithc-ca i have removed redundant `_OLD` variables.